src/css/index.css

```diff
@@ -79,6 +79,18 @@ th {
   align-self: flex-start;
   width: 100vw;
   justify-content: flex-end;
+  flex-direction: column;
+}
+
+.top-left {
+  align-self: flex-start;
+  display: flex;
+  position: absolute;
```

Comment:
Position: absolute conflicts with flex child (align-self) properties. See also: https://www.w3.org/TR/css-flexbox-1/#abspos-items

Reply:
Thanks for suggesting, looking it up.